uniffi_bindgen/src/interface/mod.rs

```diff
@@ -375,16 +375,8 @@ impl ComponentInterface {
     /// tightly with the host GC, and hence need to perform manual destruction of objects.
     pub fn item_contains_object_references(&self, item: &Type) -> bool {
         // this is surely broken for external records with object refs?
-        self.iter_types_in_item(item).any(|t| {
-            matches!(
-                t,
-                Type::Object { .. }
-                    | Type::External {
-                        kind: ExternalKind::Interface,
-                        ..
-                    }
-            )
-        })
+        self.iter_types_in_item(item)
+            .any(|t| matches!(t, Type::Object { .. }))
```

Comment:
Is this still broken with external records?  I'm not sure, but maybe the comment above can be removed.